silkworm/silkrpc/core/fee_history_oracle.cpp

```diff
@@ -52,7 +52,7 @@ void to_json(nlohmann::json& json, const FeeHistory& fh) {
     }
 }
 
-Task<FeeHistory> FeeHistoryOracle::fee_history(uint64_t newest_block, uint64_t block_count, const std::vector<std::int8_t>& reward_percentile) {
+Task<FeeHistory> FeeHistoryOracle::fee_history(BlockNum newest_block, BlockNum block_count, const std::vector<std::int8_t>& reward_percentile) {
```

Comment:
The same change should be applied also in `fee_history_oracle.hpp`

Reply:
ok